odl/contrib/mrc/README.md

```diff
@@ -0,0 +1,26 @@
+# MRC
```

Comment:
I'll see if I can find one, in particular one that can simply be downloaded by URL. Perhaps @ozanoktem has an idea?